website/src/docs/dashboard.md

```diff
@@ -252,6 +252,33 @@ It gets passed `({value, onChange}, h)` where `value` is the current value of th
 })
 ```
 
+You can also specify a function that takes a [File Object](https://uppy.io/docs/uppy/#File-Objects) and returns an array of UI field objects:
```

Comment:
Maybe add “instead of an array” or similar? Not sure if it’s clear right now that this is about `metaFields`, because the example above refers to `render`.